src/libs/sfdk/sdk.cpp

```diff
@@ -113,6 +115,16 @@ Sdk *Sdk::instance()
     return s_instance;
 }
 
+QString Sdk::osVariant()
+{
+    return Sfdk::Constants::OS_NAME;
+}
+
+QString Sdk::sdkVariant()
+{
+    return Sfdk::Constants::SDK_NAME;
```

Comment:
These are supposed to be processed with `tr()` already in the commit that introduces them.